repository.json

```diff
@@ -244,5 +244,14 @@
     "maintainer": "official",
     "displayOnWebsiteLib": true,
     "type": "data_out"
+  },
+  {
+    "name": "Pagerduty (BETA)",
+    "url": "https://github.com/PostHog/posthog-pagerduty-plugin",
+    "description": "This plugin alerts PagerDuty when a PostHog insights/trends graph goes below or above a threshold.",
+    "verified": true,
+    "maintainer": "official",
+    "displayOnWebsiteLib": true,
+    "type": "monitoring"
```

Comment:
I need to both document this **and** expand its implementation.

Currently the only accepted plugin types are `data_in`, `data_out`, and `ingestion_filtering`.

These are very much legacy and don't encompass all plugins. I'd flag this as `data_out` for now before we update https://posthog.com/plugins